src/Pos/DB/Block.hs

```diff
@@ -247,13 +248,19 @@ deleteData fp = (liftIO $ removeFile fp) `catch` handle
         | isDoesNotExistError e = pure ()
         | otherwise = throwM e
 
+dirDataPath :: MonadDB m => HeaderHash -> m FilePath
+dirDataPath (formatToString hashHexF -> fn) = gitDirDataPath fn
```

Comment:
I guess it'd be more obvious to store them by epoch index. Why did you go with hash appproach?